apps/coscrad-frontend/src/components/resources/terms/term-detail.full-view.presenter.tsx

```diff
@@ -18,11 +19,11 @@ export const TermDetailFullViewPresenter = ({
 }: ICategorizableDetailQueryResult<ITermViewModel>): JSX.Element => {
     return (
         <ResourceDetailFullViewPresenter name={name} id={id} type={ResourceType.term}>
-            <div data-testid={id} />
+            <Box data-testid={id} />
             <SingleOptionalPropertyPresenter display="Contributor" value={contributor} />
-            <div id="media-player">
-                <MediaPlayer listenMessage="Play!" audioUrl={audioURL} />
-            </div>
+            <Box id="media-player">
+                <AudioClipPlayer audioUrl={audioURL} PlayButton={DefaultPlayButton} />
```

Comment:
Let's update the API so that you don't have to pass a `PlayButton` in if you want the default.

Reply:
done